src/modules/sync/lib/convert-tree-to-sync-options.ts

```diff
@@ -9,78 +9,130 @@ type PushOptionsWithSelections = {
 		plugins?: string[];
 		themes?: string[];
 		uploads?: string[];
+		contents?: string[];
 	};
 };
 
-const isSyncOption = ( value: string ): value is SyncOption => {
-	return Object.keys( SYNC_OPTIONS ).includes( value );
+const STANDARD_CATEGORIES = [
+	{ prefix: 'plugins/', key: 'plugins' as const, option: SYNC_OPTIONS.plugins },
+	{ prefix: 'themes/', key: 'themes' as const, option: SYNC_OPTIONS.themes },
+	{ prefix: 'uploads/', key: 'uploads' as const, option: SYNC_OPTIONS.uploads },
+] as const;
+
+const CONTENTS_ROOTS = [ 'mu-plugins', 'fonts', 'languages' ] as const;
+
+const normalizePath = ( path: string ): string =>
+	path.replace( /^\/?wp-content\//, '' ).replace( /^\/+|\/+$/g, '' );
+
+const traverseSelected = (
```

Comment:
```suggestion
const iterateOverCheckedNodes = (
```
Sorry for all the nitpicks. I'm making sense of the logic in this file and commenting on things as I go. In this case, I think we can rename the function for added clarity.

Reply:
https://github.com/Automattic/studio/pull/1940/commits/1ed60953698ef49ae7f4b2792b1c769ab1dfe1d1